### ISSUE TITLE
Bug Report: current docker-php-extension-installer does not officialy support PHP 8.2

### ISSUE THREAD

**Issue by jnoordsij** [CircleCI-Public/cimg-php#239]
*Note: We also welcome PRs to fix bugs! This helps us take action faster where a bug has been identified!*

For our official CircleCI Docker Convenience Image support policy, please see [CircleCI docs](https://circleci.com/docs/convenience-images-support-policy).

This policy outlines the release, update, and deprecation policy for CircleCI Docker Convenience Images.

---

**Describe the bug**
Running the current docker-php-extension-installer script in the PHP 8.2 base image, e.g. `sudo -E install-php-extensions xdebug`, yields the following message:
```
### ERROR: Unsupported PHP version: 8.2 ###
```
Altough technically it continues and works for the cases I've found, an update to the extension would probably be appropriate.

**To Reproduce**
`docker container run --rm -it cimg/php:8.2 /bin/bash -c "sudo -E install-php-extensions xdebug"`

**Expected behavior**
No error message appears.

**Workarounds**
Ignore the error message as it appears unharmful in most cases.

**Screenshots and Build Links**
n/a

**Additional context**
With the PHP 8.3 release coming up, it might be good to update the extension in such a manner it officialy supports that as well. Given that the current origin repository supports it (see https://github.com/mlocati/docker-php-extension-installer/pull/810), getting up to date with this should be sufficient.

**Comment by matiaslauriti** [comment]
It is fully working now, using PHP `8.3`, literally executing 
```shell
sudo -E install-php-extensions xdebug
sudo docker-php-ext-enable xdebug
```

**Comment by jnoordsij** [comment]
> It is fully working now, using PHP `8.3`, literally executing
> 
> ```shell
> sudo -E install-php-extensions xdebug
> sudo docker-php-ext-enable xdebug
> ```

It is functional, but the warning still appears. So although not a large problem, the problem is still present.

**Comment by matiaslauriti** [comment]
> > It is fully working now, using PHP `8.3`, literally executing
> > ```shell
> > sudo -E install-php-extensions xdebug
> > sudo docker-php-ext-enable xdebug
> > ```
> 
> It is functional, but the warning still appears. So although not a large problem, the problem is still present.

You are correct. When I checked, the output was scrolled to the bottom, hiding the first line (the warning you are talking about)
